test/system/reports_test.rb

```diff
@@ -4,44 +4,49 @@
 
 class ReportsTest < ApplicationSystemTestCase
   setup do
-    @report = reports(:one)
+    @report = reports(:first)
+
+    visit root_url
+    fill_in 'Eメール', with: 'alice@example.com'
+    fill_in 'パスワード', with: 'password'
+    click_on 'ログイン'
+
+    click_on '日報'
   end
 
   test 'visiting the index' do
     visit reports_url
-    assert_selector 'h1', text: 'Reports'
+    assert_selector 'h1', text: '日報の一覧'
   end
 
   test 'should create report' do
     visit reports_url
-    click_on 'New report'
+    click_on '日報の新規作成'
 
-    fill_in 'Content', with: @report.content
-    fill_in 'Title', with: @report.title
-    fill_in 'User', with: @report.user_id
-    click_on 'Create Report'
+    fill_in 'タイトル', with: @report.title
```

Comment:
```suggestion
    fill_in 'タイトル', with: "..."
```

変数よりも文字列や数値をベタ書きした方がテストコードが読みやすいです

https://speakerdeck.com/jnchito/number-vstat?slide=42